<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_LevelSettings</class>
 <widget class="QDialog" name="GUI_LevelSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>380</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>450</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New level</string>
  </property>
  <widget class="QDialogButtonBox" name="confirmation_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>166</width>
     <height>21</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="levelGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Level file</string>
   </property>
   <widget class="QLineEdit" name="pathLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selectPathBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="settingsGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>431</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Settings</string>
   </property>
   <widget class="QGroupBox" name="levelSizeGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Level size</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>391</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="layout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="levelWidth_label">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="levelHeight_label">
        <property name="text">
         <string>Height: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="levelWidth_spinBox">
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>8192</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="levelHeight_spinBox">
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>8192</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gridGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Grid size</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>391</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="layout">
      <item row="0" column="0">
       <widget class="QLabel" name="gridWidth_label">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="gridHeight_label">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="gridWidth_spinBox">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>4</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="gridHeight_spinBox">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>4</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
